wireless.lua

```diff
@@ -47,6 +47,39 @@ local function draw_signal(level)
     return img
 end
 
+function net_stats(card,which)
+local prefix = {
+	[0] = "",
+	[1] = "K",
+	[2] = "M",
+	[3] = "G",
+	[4] = "T"
+}
+local function readAll(file)
+  local f = assert(io.open(file, "rb"))
+  local content = f:read()
+  f:close()
+  return content
+end
+local function round(num, numDecimalPlaces)
+  local mult = 10^(numDecimalPlaces or 0)
+  return math.floor(num * mult + 0.5) / mult
+end
+  if (which == "d") then
+    f = readAll("/sys/class/net/" .. card .. "/statistics/rx_bytes")
+  else if (which == "u") then
+    f = readAll("/sys/class/net/" .. card .. "/statistics/tx_bytes")
+    end
+  end
+	local count = 0
+	local stat = tonumber(f)
+	while (stat > 999) do
```

Comment:
Why not 1024?

Reply:
carried it over from the ugly hacky script, fixed now